lagom/shopping-cart-scala/shopping-cart/src/main/resources/cinnamon.conf

```diff
@@ -22,32 +22,27 @@ cinnamon {
 
   chmetrics.reporters += "console-reporter"
 
+  #
+  # Uncomment if you use Prometheus as a backend.
+  # See https://developer.lightbend.com/docs/telemetry/current/sandbox/prometheus-sandbox.html
+  #
+  # prometheus.exporters += http-server
+
   akka.actors {
-    default-by-class {
```

Comment:
You're right.

There's also this documentation:

https://developer.lightbend.com/docs/telemetry/current/instrumentations/lagom/lagom-projection.html

So there's 3 different options. I think the default one is the one that makes more sense since this is a general-purpose sample (not projections-specific).